specs/core/0_beacon-chain.md

```diff
@@ -1038,7 +1036,7 @@ def initiate_validator_exit(state: BeaconState, index: ValidatorIndex) -> None:
 
     # Compute exit queue epoch
     exit_epochs = [v.exit_epoch for v in state.validators if v.exit_epoch != FAR_FUTURE_EPOCH]
-    exit_queue_epoch = max(exit_epochs + [compute_activation_exit_epoch(get_current_epoch(state))])
+    exit_queue_epoch = max(exit_epochs, default=compute_activation_exit_epoch(get_current_epoch(state)))
```

Comment:
Looks like this is a not cosmetic change, but a behavior one. 

Say we have `exit_epochs=[100, 101, 102]` and `compute_activation_exit_epoch(get_current_epoch(state)))` evaluates `105`.
- `max(exit_epochs + [compute_activation_exit_epoch(get_current_epoch(state))])` evaluates `105`, and
- `max(exit_epochs, default=compute_activation_exit_epoch(get_current_epoch(state)))` evaluates `102`

The old expression wants a min of max, while the new expression wants a default value when `exit_epochs` is empty. That results in these behaviors:

- The old one says "' Even' if no one is exiting soon, you can't exit immediately", and 
- The new one says "If no one is exiting soon, you can't exit immediately. But if someone is, you can exit with them at the same epoch."

I think the former is what we want.

Reply:
Good catch :+1: 
And also note that `exit_epochs` may contain already-exited validators, so `default` will effectively not happen after the first exit. So it looks like it was completely broken :(